packages/yoroi-extension/app/containers/wallet/voting/VotingPage.stories.js

```diff
@@ -67,6 +67,22 @@ const defaultProps: ({|
     delegation: {
       getDelegationRequests: walletLookup([request.wallet]).getDelegation,
     },
+    substores: {
+      ada: {
+        votingStore: {
+          catalystRoundInfo: {
+            currentFund: {
```

Comment:
I added this because `flow`  complaining as the `catalystRounInfo` need a default value